client/src/App.js

```diff
@@ -1,7 +1,7 @@
 import AppContainer from "./components/app-container/appContainer";
 import "./App.css";
 function App() {
-  return (
+
```

Comment:
Is it a mistake? I don't think it works with return

Reply:
Was a mistake, I've corrected it